experiments/dataSources/HubSpotREST/README.md

```diff
@@ -0,0 +1,20 @@
+# Skuid + HubSpot REST API
+
+## Connect to your HubSpot instance
+[HubSpot REST API documentation](https://developers.hubspot.com/docs/api/overview)
+
+### Data Source
+* Go to Data Sources and create a new data source with these attributes:
+    * For Type, choose REST
+    * For URL / Endpoint, use `https://api.hubapi.com`
+    * For Common request headers, for field, use Authorization, and for value, use `Bearer <your API Key>`
```

Comment:
Look at the format of documenting Common Request headers here:  https://github.com/skuid/skuid-labs/tree/master/experiments/dataSources/airtable.  I think it could be a bit clearer.